### PR TITLE
Add scaling it test of manual mode

### PR DESCRIPTION
**Pull request by azexcy**
Changes proposed in this pull request:
- Add scaling manual  mode IT test

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/BaseScalingITCase.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            }
+        }
+        assertThat(actualStatusMap.values().stream().filter(StringUtils::isNotBlank).collect(Collectors.toSet()).size(), is(1));
+        connection.createStatement().execute(String.format(getCommonSQLCommand().getStopScalingSourceWriting(), jobId));
+        ResultSet checkScalingResult = connection.createStatement().executeQuery(String.format(commonSQLCommand.getCheckScalingDataMatch(), jobId));
+        while (checkScalingResult.next()) {
+            assertTrue(checkScalingResult.getBoolean(5));
+        }
+        connection.createStatement().execute(String.format(getCommonSQLCommand().getApplyScaling(), jobId));
+        ResultSet previewResult = connection.createStatement().executeQuery(getCommonSQLCommand().getPreviewSelectOrder());
```
Comment (sandynz) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/BaseScalingITCase.java:181]: `ResultSet` should be closed.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/BaseScalingITCase.java [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        List<String> actualTargetNodes = Lists.newLinkedList();
+        while (previewResult.next()) {
+            actualTargetNodes.add(previewResult.getString(1));
+        }
+        assertThat(actualTargetNodes, is(Lists.newArrayList("ds_2", "ds_3", "ds_4")));
+    }
+    
+    /**
+     * Initialize table data.
+     * @param connection proxy database connection
```
Comment (sandynz) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/BaseScalingITCase.java:191]: New line is needed between method javadoc and param doc.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ * MySQL auto rule scaling test case.
+ */
```
Comment (sandynz) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java:48]: `auto` should be `manual`, the same class name.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+@Slf4j
+public final class MySQLManualScalingCase extends BaseScalingITCase {
```
Comment (sandynz) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java:50]: Could it be shared with PostgreSQL?

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/common/command.xml [hunk 5]
```diff
@@ -17,20 +17,86 @@
 
 <command>
     <create-database>
-        CREATE DATABASE sharding_db;
+        CREATE DATABASE sharding_db
     </create-database>
```
Comment (sandynz) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/common/command.xml:21]: The last `;` could be kept, the same as there're multiple SQLs.

**Review by sandynz — APPROVED** [review]

**Comment by sandynz** [comment]
The review suggestion will be done in later PR.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java (hunk 4 above)
Comment (azexcy) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java:50]: They are a little different in creating table statements，i will extract public parts at BaseScalingITCase.class

**Pull request merged**